carbonserver/carbonserver/api/routers/emissions.py

```diff
@@ -45,7 +46,16 @@ def add_emission(
     emission_service: EmissionService = Depends(
         Provide[ServerContainer.emission_service]
     ),
+    project_token_service: ProjectTokenService = Depends(
+        Provide[ServerContainer.project_token_service]
+    ),
+    x_project_token: str = Header(None),  # Capture the x-project-token from the headers
```

Comment:
`x_api_token` instead ?
it's a pretty standard header, we can use that instead of cooking our own variant 
the token's scope might be limited, it's still an api token

Reply:
Done, see [ba08bd4](https://github.com/mlco2/codecarbon/pull/624/commits/ba08bd46a5d0d5c7f2e67e50a8a5a8c10213bbc3)